### ISSUE TITLE
FloatingPanel - Mainstage is before storytext in document order

### ISSUE THREAD

**Issue by regan-sarwas** [regan-sarwas/map-journal-storytelling-template-js#9]
This can cause some confusing tab order (especially compared with the side panel).  The main stage should be a sidebar to the active story section, and be focusable by choice at the end of each section.

This generally works, when using the navigation controls, but not when simply tabbing through the document.

Not fixing because the floating panel has bigger issues.
